test-app/src/main/java/org/readium/r2/testapp/domain/model/Highlight.kt

```diff
@@ -85,7 +84,15 @@ data class Highlight(
     enum class Style(val value: String) {
         HIGHLIGHT("highlight"), UNDERLINE("underline");
 
-        companion object : MapWithDefaultCompanion<String, Style>(values(), Style::value, HIGHLIGHT)
+        companion object {
```

Comment:
Nothing per se, but it's supposed to be an internal Readium helper class. So I don't think we should encourage its use in the Test App as it would need to follow semantic versioning. I'll add an `@InternalReadiumApi` annotation to it.

Reply:
Ok. You could have just copy it into the testapp...